addons/block_code/simple_spawner/simple_spawner.gd

```diff
@@ -79,7 +79,7 @@ func spawn_once():
 	if scenes.size() == 0:
 		return
 
-	_spawned_scenes = _spawned_scenes.filter(is_instance_valid)
+	_spawned_scenes = _spawned_scenes.filter(func(instance): return is_instance_valid(instance))
```

Comment:
I try to understand the `func()` and `instance`.  Then, I found [Godot 4 lambda functions](https://www.youtube.com/watch?v=N4yX8lbBRWY).